components/navbar/navbarPage.html

```diff
@@ -180,38 +180,48 @@ <h3 class="dui-util-fw-bld dui-util-clr-prim-p2">
                                 <line x1="21" y1="21" x2="15" y2="15" />
                             </svg>
                         </i>
-                        <input
-                            class="dui-inp-txt__input dui-inp-txt__input--search dui-util-txt-sm dui-util-bdr-radi-5px-s reset-input-inherit-parent"
-                            type="text" placeholder="Search">
+                        <label>
+                            <input
+                                class="dui-inp-txt__input dui-inp-txt__input--search dui-util-txt-sm dui-util-bdr-radi-5px-s reset-input-inherit-parent"
+                                type="text" placeholder="Search">
+                        </label>
                     </div>
 
                     <div class="dui-nav__actions">
-                        <!-- Login Button -->
-                        <a class="dui-nav__login-btn dui-link dui-link--primary-p2 dui-util-txt-sm dui-util-spc-pad-0_8rem-xs dui-util-fw-bld"
-                            href="#">
-                            Login
-                        </a>
+                        <div class="dui-nav__auth-actions">
+                            <!-- SignUp Button -->
+                            <a class="dui-nav__signup-btn dui-link dui-link--primary dui-util-txt-sm dui-util-spc-pad-0_8rem-xs dui-util-fw-bld"
+                                href="#" style="margin-right: 2rem;">
+                                Sign Up
+                            </a>
 
-                        <div>
-                            <button class="dui-nav__wish-lst-btn dui-btn reset-button-inherit-parent">
-                                <div class="dui-badge dui-badge--round-ele-0_5-top-0_5-right">
-                                    <div>
-                                        0
-                                    </div>
+                            <!-- Login Button -->
+                            <a class="dui-nav__login-btn dui-link dui-link--primary dui-util-txt-sm dui-util-spc-pad-0_8rem-xs dui-util-fw-bld"
+                                href="#">
+                                Login
+                            </a>
+                        </div>
+
+                        <!-- <div> -->
+                        <a class="dui-nav__wish-lst-btn dui-util-disp-inline-block" href="#">
+                            <div class="dui-badge dui-badge--round-ele-0_5-top-0_5-right">
+                                <div>
+                                    0
                                 </div>
```

Comment:
fix: you can use p tag here

Reply:
Intentionally gave it div, as it can contain any element ex: svg, or img